<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pyLottoSimu</string>
  </property>
  <widget class="QWidget" name="widget_lottery_numbers">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>480</width>
     <height>320</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>480</width>
     <height>320</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>lottery numbers</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#Rahmen { border-image: url(:/lottokugel.png)}
#QLabel {	background-color: rgb(220, 220, 220)}
QLabel#label_zahl, QLabel#label_zahl_2 {	background-color: none}</string>
   </property>
   <widget class="QFrame" name="Rahmen">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>320</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>441</width>
       <height>16</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string extracomment="output field">output field</string>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_Lottozahlen">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>441</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>lottery numbers</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>351</width>
       <height>33</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Numbers:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbox_drawn_numbers">
        <property name="toolTip">
         <string>drawn numbers</string>
        </property>
        <property name="statusTip">
         <string>drawn numbers</string>
        </property>
        <property name="whatsThis">
         <string extracomment="drawn numbers">drawn numbers</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
        <property name="value">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>out of</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbox_from_a_set_of">
        <property name="toolTip">
         <string>from a set of</string>
        </property>
        <property name="statusTip">
         <string>from a set of</string>
        </property>
        <property name="whatsThis">
         <string extracomment="from a set of">from a set of</string>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="minimum">
         <number>20</number>
        </property>
        <property name="maximum">
         <number>90</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>49</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="btn_random_numbers">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>260</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>random numbers</string>
     </property>
     <property name="statusTip">
      <string>random numbers</string>
     </property>
     <property name="text">
      <string>random numbers</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AusgfeldLeeren">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>180</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>output field empty</string>
     </property>
     <property name="text">
      <string>output field empty</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_zahl">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>121</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>32</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Text</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_start">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>110</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>start draw</string>
     </property>
     <property name="statusTip">
      <string>start draw</string>
     </property>
     <property name="text">
      <string>start draw</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Speed</string>
     </property>
     <property name="statusTip">
      <string>Speed</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>99</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="sliderPosition">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>true</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_Geschwindigkeit">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>80</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Speed</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_zahl_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>50</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>26</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Text</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionBeenden"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionInfo"/>
   </widget>
   <widget class="QMenu" name="menuModus">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="actionLottosimulation"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuModus"/>
   <addaction name="menuHilfe"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBeenden">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionZufallsgenerator">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>random numbers</string>
   </property>
  </action>
  <action name="actionLottosimulation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Lotto simulation</string>
   </property>
  </action>
  <action name="actionHilfe">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="lottokugeln.qrc"/>
 </resources>
 <connections/>
</ui>
